.github/workflows/tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'dev'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
```

Comment:
The project structure, including the Github workflows, were based on django-components repo.